sycamore/transforms/extract_table.py

```diff
@@ -39,6 +39,8 @@ def bbox_to_coord(bbox):
             if table.title:
                 element.text_representation = table.title.text + "\n"
                 element.properties["boxes"].append(bbox_to_coord(table.title.bbox))
+            else:
+                element.text_representation = ""
```

Comment:
yes, as you mentioned in issue, it's also worthwhile to initialize to empty string.

Reply:
I'm happy to initialize to empty string, but I worry that some code may depend on the difference between None and "" for some logic.  It's a bigger, more general change.